bool                        changed
geometry_msgs/PoseStamped   goal                                                       
